extensions/kafka/ConsumeKafka.h

```diff
@@ -219,9 +217,13 @@ class ConsumeKafka : public KafkaProcessorBase {
 
   ADD_COMMON_VIRTUAL_FUNCTIONS_FOR_PROCESSORS
 
-  explicit ConsumeKafka(std::string_view name, const utils::Identifier& uuid = utils::Identifier()) :
+  explicit ConsumeKafka(const std::string_view name, const utils::Identifier& uuid = utils::Identifier()) :
```

Comment:
@szaszm beat me to it with a nice explanation :+1:, but yeah afaik the constness of the argument only matters in the definition for the parameters mutability, and it has no effect in the declaration.
Here I've added it because we have the declaration and the definition in the same place.